README.md

```diff
@@ -176,18 +176,47 @@ App|Description
 
 ### USB Device
 
+#### TinyUSB Examples 
+
+All but one USB device examples come directly from the TinyUSB device examples directory [here](https://github.com/hathach/tinyusb/tree/master/examples/device).
```

Comment:
Would "All but one of the USB device examples" be better grammatically?

Reply:
fixed